Task 1/2019201419王成瑞/main.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include <stdint.h>
+#include "double.h"
```

Comment:
In the next serveral lessons you will be told that header files should usually contain only declarations NOT definitions. If you like separate your code to multiple files (yes, it's a good practice), you can use multiple `.c` files.